README.md

```diff
@@ -11,41 +11,51 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image using the docker daemon use the `image` parameter:
```

Comment:
I think we want to be clear that the Docker daemon isn't doing the scanning. Syft is _fetching the image_ from the Docker daemon in this scenario

Reply:
"To scan an image _from_ the local docker daemon"? Happy for any wording suggestions here...